creator-node/test/StateMonitoringQueue.test.js

```diff
@@ -71,6 +72,26 @@ describe('test StateMonitoringQueue initialization and logging', function () {
       })
   })
 
+  it("doesn't queue or process any jobs when rate limit jobsPerInterval is set to 0", async function () {
```

Comment:
amazing - doesn't test that job eventually gets processed after rate limit interval resets, but feels weird to write a unit test for a core dependency feature

we should maybe think about upgrading bull to latest version tho (separate)

Reply:
we actually upgraded to the most recent version a couple weeks ago, but I'm down to do it again and to try upgrading to BullMQ soon

Reply:
made a Linear card to follow up: https://linear.app/audius/issue/CON-175/upgrade-bull-to-bullmq-in-content-node